the-metalworks/cosmere-rpg
# chore(release notes): separate major/minor and patch release notes

**Type**  
What type of pull request is this? (e.g., Bug fix, Feature, Refactor, etc.)


**Description**  
This PR separates major/minor release notes from patch release notes so we can show the appropriate one to the user (depending on their previous version).

**Related Issue**  
Closes #172 

**How Has This Been Tested?**  
1. Update system version to `0.2.1` locally (requires Foundry restart)
2. Load the world
3. See the patch notes 
4. Update the `latestVersion` system setting to `0.1.2`
5. Reload the world
6. See the minor release notes

**Checklist:**
